src/index.js

```diff
@@ -68,22 +67,11 @@ const ApifyClient = function (options = {}) {
         return (callOpts, callback) => {
             const mergedOpts = Object.assign({}, instanceOpts, callOpts);
 
+            // Remove traling forward slash from baseUrl.
+            if (mergedOpts.baseUrl.substr(-1) === '/') mergedOpts.baseUrl = mergedOpts.baseUrl.slice(0, -1);
```

Comment:
it's because user can override every parameter in method call also

apifyClient.crawlers.getCrawler({ crawlerId: 'ljknjk', baseUrl: 'urlWithTrailingSlash/' )